### PR TITLE
Enable linter for duplicated imports

### PR DESCRIPTION
**Pull request by shafeeqes**
**How to categorize this PR?**

/area quality
/kind enhancement

**What this PR does / why we need it**:
Enable linter for duplicated imports.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
/cc @rfranzke 

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ issues:
   - indent-error-flow # if block ends with a return statement, so drop this else and outdent its block
   - "exported: (type|func) name will be used as .* by other packages, and that stutters;"
   - unused-parameter # parameter '.*' seems to be unused, consider removing or renaming it as _
+  - 'duplicated-imports: Package "embed" already imported' # Allow duplicated import for "embed" package
```
Comment (rfranzke) [inline .golangci.yaml]: Why do we allow this? 👀

**Review by rfranzke — COMMENTED** [review]
Nice findings 😄

File: .golangci.yaml (hunk 1 above)
Comment (shafeeqes) [reply .golangci.yaml]: Hmm, not sure. It looks like we can remove the empty import in `pkg/provider-local/charts/charts.go`. Adapted.

Comment (shafeeqes) [reply .golangci.yaml]: I was thinking if we had a variable like,
```
	//go:embed images.yaml
	imagesYAML  string
```
as well, then, the empty import is required. But maybe not.

Comment (rfranzke) [reply .golangci.yaml]: OK, but how is this related to allowing duplicate embed import? 😄

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**